ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -134,6 +135,46 @@ EOF
 EOF
     fi
   fi
+
+  # overide junit file to insert post check
+  # once found installation post check failure
+  if test -f "${SHARED_DIR}/install-post-check-status.txt"
+  then
+    set +o errexit
+    grep -q "^$EXIT_CODE_POSTCHECK$" "${SHARED_DIR}/install-post-check-status.txt"
+    INSTALL_POSTCHECK_FAILURE=$((1-$?))
+    [ "$INSTALL_POSTCHECK_FAILURE" = 1 ] && INSTALL_POSTCHECK_FAILURE=2
```

Comment:
Here can we directly use the return value of `grep` command for if condition instead of introducing `INSTALL_POSTCHECK_FAILURE` ?

Reply:
Here need to use `INSTALL_POSTCHECK_FAILURE` to count failed test cases, and set value to attribute `failures` in junit_install.xml.
If post check failed, two test cases should be failed, "install should succeed: post check" and "install should succeed: overall"
or changed as below?
```
if grep -q "^$EXIT_CODE_POSTCHECK$" "${SHARED_DIR}/install-post-check-status.txt"; then
    INSTALL_POSTCHECK_FAILURE=2
else
    INSTALL_POSTCHECK_FAILURE=0
fi
```

Reply:
> If post check failed, two test cases should be failed, "install should succeed: post check" and "install should succeed: overall"

Ah, got it.

> changed as below

okay, this change sounds be easier to understand.

Reply:
updated